src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionUpdateActionUtils.java

```diff
@@ -42,13 +45,33 @@
         @Nonnull final AttributeDefinitionDraft newAttributeDefinitionDraft) {
 
         final List<UpdateAction<ProductType>> updateActions = filterEmptyOptionals(
-            buildChangeLabelUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
-            buildSetInputTipUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
-            buildChangeIsSearchableUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
-            buildChangeInputHintUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
-            buildChangeAttributeConstraintUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft)
+                buildChangeLabelUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+                buildSetInputTipUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+                buildChangeIsSearchableUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+                buildChangeInputHintUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+                buildChangeAttributeConstraintUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft)
         );
 
+        updateActions.addAll(buildEnumUpdateActions(oldAttributeDefinition, newAttributeDefinitionDraft));
+        return updateActions;
+    }
+
+    /**
+     * Compares all the {@link EnumValue} and {@link LocalizedEnumValue} values of an {@link AttributeDefinition} and
+     * an {@link AttributeDefinitionDraft} and returns a list of {@link UpdateAction}&lt;{@link ProductType}&gt; as a
+     * result. If both the {@link AttributeDefinition} and the {@link AttributeDefinitionDraft} have identical
+     * enum values, then no update action is needed and hence an empty {@link List} is returned.
+     *
+     * @param oldAttributeDefinition      the attribute definition which should be updated.
+     * @param newAttributeDefinitionDraft the new attribute definition draft where we get the new fields.
+     * @return A list with the update actions or an empty list if the attribute definition enums are identical.
+     */
+    @Nonnull
+    static List<UpdateAction<ProductType>> buildEnumUpdateActions(
```

Comment:
? @ahmetoz

Reply:
Yes, you should put the commit, because it was not clear if you addressed or did not agree with this suggestion :D

Reply:
Gotcha! I did not think like this. But is there any other option? it looks like I committed your suggestion and it become resolved automatically because commit suggestion is disabled.